pkg/virt-controller/services/template.go

```diff
@@ -1377,11 +1377,18 @@ func getMemoryOverhead(vmi *v1.VirtualMachineInstance) *resource.Quantity {
 	// static overhead for IOThread
 	overhead.Add(resource.MustParse("8Mi"))
 
-	// Add video RAM overhead
-	if domain.Devices.AutoattachGraphicsDevice == nil || *domain.Devices.AutoattachGraphicsDevice == true {
+	// VGA device is not support by buildin qemu-kvm in virt-launcher image for aarch64
+	// Add video RAM overhead when GOARCH is not aarch64
+	if runtime.GOARCH != "arm64" && (domain.Devices.AutoattachGraphicsDevice == nil || *domain.Devices.AutoattachGraphicsDevice == true) {
 		overhead.Add(resource.MustParse("16Mi"))
 	}
```

Comment:
Can we avoid accounting for this special condition here?

My expectation is that autoattachGraphicsDevice would be defaulted to false for arm VMs in the mutation webhook when the `virt` machine type is used... or another option would be that we'd not accept a vm with an arm machine type unless autoattachGraphisDevice